app/views/shared/_messages.html.erb

```diff
@@ -1,11 +1,12 @@
-<% flash.each do |type, msg| %>
-  <% if type == "error" %>
-    <div class="alert alert-danger" role="alert">
-      <%= msg %>
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      The form contains <%= pluralize(@user.errors.count, "error") %>.
     </div>
-  <% else %>
-    <div class="alert alert-info">
-      <%= msg %>
-    </div>
-  <% end %>
```

Comment:
Chỗ nì sao em lại sửa vậy???

Reply:
tai vi may cai flash cu dang k show up on the redirected screen sau khi update xong user a the nen la phai de flash vao application.html.erb; cai nay de show errors trong cai form day luon k redirect th a